pkg/util/slice/slice.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package slice
+
+import (
+	"fmt"
+	"reflect"
+)
+
+func ItemExists(slice interface{}, item interface{}) (bool, error) {
+	s := reflect.ValueOf(slice)
+
+	if s.Kind() != reflect.Slice {
+		return false, fmt.Errorf("Not a slice")
+	}
+
+	for i := 0; i < s.Len(); i++ {
+		if s.Index(i).Interface() == item {
```

Comment:
here also you are comparing the 2 different interfaces if we know what type it contains then might be good to use that.